.github/workflows/update_sdk_methods.py

```diff
@@ -50,6 +50,7 @@
                      the complete data object from parse() to /tmp/update_sdk_methods_debug.txt. \
                      Also prints high-level status updates to STDOUT. \
                      Deletes previous debug file when run again.")
+parser.add_argument('-c', '--coverage', action='store_true', help="Fails the build if any methods are unused.")
```

Comment:
What does "unused" mean? Seems like this should say "not documented"?

Reply:
methods in the sdk docs that didn't get used / put into generated files

Reply:
I just think it's not the clearest term for this